src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -2571,6 +2576,52 @@ internal static ByteStringContext.Scope GenerateBucketAndEtagIndexKeyForTombston
             return GenerateBucketAndEtagIndexKey(context, idIndex: (int)TombstoneTable.LowerId, etagIndex: (int)TombstoneTable.Etag, ref tvr, out slice);
         }
 
+        // TODO : use a dedicated attribute
+        internal static void UpdateBucketStatsForDocument(Transaction tx, Slice key, int oldSize, int newSize)
+        {
+            UpdateBucketStats(tx, key, oldSize, newSize, isDocument: true);
+        }
+
+        internal static void UpdateBucketStats(Transaction tx, Slice key, int oldSize, int newSize)
+        {
+            UpdateBucketStats(tx, key, oldSize, newSize, isDocument: false);
+        }
+
+        private static void UpdateBucketStats(Transaction tx, Slice key, int oldSize, int newSize, bool isDocument)
```

Comment:
What happens to the size with documents compression?

Reply:
not working as expected, we count the Raw size and not the Compressed size
added a test (skipped for now) and created an issue : https://issues.hibernatingrhinos.com/issue/RavenDB-19530/